.gitignore

```diff
@@ -54,6 +54,7 @@ playground.xcworkspace
 # Carthage/Checkouts
 
 Carthage/Build
+Baymax.framework.zip
```

Comment:
Generic question for you for all our projects: Why don't we ignore `Carthage`, the checkouts only add to the size of the remote repo.